### DISCUSSION TITLE
Loss of secondary sampling image metadata?

### DISCUSSION THREAD

**Discussion by afbagwell** [vladmandic/sdnext#2097] [General]
I've found that many of my images going back several weeks don't have the secondary sampler and steps recorded in their metadata. If I load them into "From Text", the secondary sampler remains on the default or whatever it was last manually set to instead of what was used to originally produce the image. It looks like this change happened sometime around the weekend of Aug 10, but not certain what version since I was out of town that weekend and skipped about four days worth of updates.

-A

**Comment by Tillerz** [comment]
That just got fixed yesterday. See https://github.com/vladmandic/automatic/issues/2078#event-10243951962

**Comment by afbagwell** [comment]
Thanks for fixing this!
